src/server/database_read.py

```diff
@@ -0,0 +1,29 @@
+
+
+class DatabaseReader:
```

Comment:
This is the type of thing (wrapper object) I'm looking for to be passed down from the top (exportmanager).

Reply:
Yea it's pretty nice, but also can be a lot of work to make a wrapper just for the purposes of making it read only. I'm trying to find a quicker way to do this rather than having change all the database calls to yet another class.

Reply:
Though with such a wrapper, we can do more than read-only. We can stub the entire db for testing.

Reply:
Should it be a general RaceData class and include both getters and setters that could be replaced throughout the codebase? Or does that just reinvent database queries for little benefit?